pkg/kubelet/metrics/metrics.go

```diff
@@ -556,6 +579,10 @@ func Register(collectors ...metrics.StableCollector) {
 		}
 		legacyregistry.MustRegister(RunPodSandboxDuration)
 		legacyregistry.MustRegister(RunPodSandboxErrors)
+		if utilfeature.DefaultFeatureGate.Enabled(features.CPUManager) {
```

Comment:
On another PR you mentioned that we should remove these feature gate checks when moving to GA. Is that also valid here?

Reply:
yes, I'm doing this in the graduation PR (https://github.com/kubernetes/kubernetes/pull/113018/commits)